logs_sdk/lib/opentelemetry/sdk/logs/log_record_data.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module SDK
+    module Logs
+      # LogRecordData is a Struct containing {LogRecord} data for export.
+      LogRecordData = Struct.new(:timestamp,             # optional Integer nanoseconds since Epoch
+                                 :observed_timestamp,    # Integer nanoseconds since Epoch
+                                 :trace_id,              # optional String (16-byte binary)
+                                 :span_id,               # optional String (8 byte binary)
```

Comment:
The other comments use a `-`:

```suggestion
                                 :span_id,               # optional String (8-byte binary)
```